ci/build.ps1

```diff
@@ -113,16 +87,16 @@ elseif ($compiler -eq 'MSVC') {
 
 if (-not $NoTests) {
   # Setup python (use AppVeyor system python)
-  C:\Python27\python.exe -m pip install pynvim ; exitIfFailed
-  C:\Python35\python.exe -m pip install pynvim ; exitIfFailed
+
+  C:\hostedtoolcache\windows\Python\2.7.18\x64\python.exe -m pip install pynvim ; exitIfFailed
```

Comment:
Might be better to use [Get-ToolcacheToolDirectory](https://github.com/actions/virtual-environments/blob/fc8402e007bc3b7ae4a449edd57dee7a151a331e/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L297-L301) or `$env:AGENT_TOOLSDIRECTORY`.

Reply:
I tried and found that they are not defined in the virtual environment. Isn't this for use when creating virtual images? If anyone is an expert in GHA and knows a good way to get these paths from within a virtual environment, please let me know.